package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is this in the root directory of the project for some reason? It looks like you installed axios in the wrong directory.